gamerun.cc

```diff
@@ -24,10 +24,12 @@ int gameRun(Observer *white, Observer *black, ChessGame *gameBoard) {
                     toReturn = 0;               //no legal moves remaining
                     cout << "No move available due to Stalemate!" << endl;
                 }
-            } else {                            //status is just 1
+            } else if (status == 1) {                            //status is just 1
                 white->notify();
                 gameBoard->switchTurn();
                 turn = gameBoard->getPlayerTurn();
+            } else {
+                return -9999; //return code for incomplete game
```

Comment:
Make this `toReturn = -9999` since the program stops anyway after that and returns it.